addon/utils/assign.js

```diff
@@ -1,13 +1,6 @@
 // @flow
 
-import { assign as EmberAssign } from '@ember/polyfills';
```

Comment:
I might be missing some point, but `Ember.assign` [falls back](https://github.com/emberjs/ember.js/blob/master/packages/%40ember/polyfills/lib/assign.ts#L52) to `Object.assign` if it's available.